README.md

```diff
@@ -1,8 +1,13 @@
-# Kotlin LSIF indexer 
-![status: development](https://camo.githubusercontent.com/27137af2d15ae2e5af54fb7ec1081e001370f021901724d630e23003990b08c5/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f7374617475732d646576656c6f706d656e742d79656c6c6f773f7374796c653d666c6174)
+# Kotlin LSIF support
 
-This project implements a Kotlin compiler plugin that generates an LSIF index from a project, using [SemanticDB](https://scalameta.org/docs/semanticdb/guide.html) as an intermediary format.   
+This project implements a Kotlin compiler plugin that generates an LSIF index
```

Comment:
I know this was originally mine, do we want to remove the part where it says it generates an LSIF index when this technically only emits semanticdb? :P

Reply:
Better now?